hepcrawl/spiders/oup_spider.py

```diff
@@ -192,13 +193,23 @@ def download_files_from_ftp(self, ftp_folder):
                 file_name = '%s_%s' % (
                     filename_prefix, os.path.basename(file_path))
                 local_filename = os.path.join(self.target_folder, file_name)
-                host.download(file_path, local_filename)
+                
+                try:
+                    host.download(file_path, local_filename)
+                except:
```

Comment:
Just a general comment :) It's not a good practice not to have an exception type, if we want to catch any exception (which is also not recommended) we should should use the following:

``` python
try:
...
except Exception:
...
```